.github/workflows/build.yml

```diff
@@ -13,11 +13,16 @@ jobs:
   build:
 
     runs-on: ubuntu-latest
-    
+
     steps:
     - uses: actions/checkout@v2
     - name: Set up Python 3.8
       uses: actions/setup-python@v2
       with:
         python-version: 3.8
     - uses: pre-commit/action@v2.0.0
+    - name: Run Tests
+      run: |
+        pip install --upgrade pip
+        pip install -e ".[dev]"
```

Comment:
doesn't matter right now but maybe at some point we should put this in a separate step prior to pre-commit so no installations are repeated